components/eventing-controller/controllers/subscriptionv2/jetstream/test/reconciler_integration_test.go

```diff
@@ -514,88 +516,86 @@ func TestChangeSubscription(t *testing.T) {
-			reconcilertestingv2.EventuallyUpdateSubscriptionOnK8s(ens.Ctx, ens.TestEnsemble,
+			require.NoError(t, EventuallyUpdateSubscriptionOnK8s(jsTestEnsemble.Ctx, jsTestEnsemble.Ensemble,
 				sub, func(sub *eventingv1alpha2.Subscription) error {
 					tc.changeSubscription(sub)
-					return ens.K8sClient.Update(ens.Ctx, sub)
-				})
+					return jsTestEnsemble.K8sClient.Update(jsTestEnsemble.Ctx, sub)
+				}))
 
 			// then
 			t.Log("testing the k8s subscription")
-			reconcilertestingv2.TestSubscriptionOnK8s(ens.TestEnsemble, sub, tc.wantAfter.K8sSubscription...)
+			CheckSubscriptionOnK8s(g, jsTestEnsemble.Ensemble, sub, tc.wantAfter.K8sSubscription...)
 
 			t.Log("testing the k8s events")
-			reconcilertestingv2.TestEventsOnK8s(ens.TestEnsemble, tc.wantAfter.K8sEvents...)
+			CheckEventsOnK8s(g, jsTestEnsemble.Ensemble, tc.wantAfter.K8sEvents...)
 
 			t.Log("testing the nats subscriptions")
 			for eventType, matchers := range tc.wantAfter.NatsSubscriptions {
-				testSubscriptionOnNATS(ens, sub, eventType, matchers...)
+				testSubscriptionOnNATS(g, sub, eventType, matchers...)
 			}
 
 			t.Log("testing the deletion of the subscription")
-			testSubscriptionDeletion(ens, sub)
+			testSubscriptionDeletion(g, sub)
 
 			t.Log("testing the deletion of the NATS subscription(s)")
 			for _, filter := range sub.Spec.Types {
-				ensureNATSSubscriptionIsDeleted(ens, sub, filter)
+				ensureNATSSubscriptionIsDeleted(g, sub, filter)
 			}
 		})
 	}
-	t.Cleanup(ens.Cancel)
 }
 
 // TestEmptyEventTypePrefix tests if a subscription is reconciled properly if the EventTypePrefix is empty.
-func TestEmptyEventTypePrefix(t *testing.T) {
-	ens := setupTestEnsemble(t)
-	defer cleanupResources(ens)
+func Test_EmptyEventTypePrefix(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
 
 	// when
-	sub := reconcilertestingv2.CreateSubscription(ens.TestEnsemble,
+	sub := CreateSubscription(t,
+		jsTestEnsemble.Ensemble,
 		testingv2.WithTypeMatchingExact(),
 		testingv2.WithSourceAndType(emptyEventSource, testingv2.OrderCreatedEventTypePrefixEmpty),
-		testingv2.WithSinkURLFromSvc(ens.SubscriberSvc),
+		testingv2.WithSinkURLFromSvc(jsTestEnsemble.SubscriberSvc),
 	)
 
 	// then
-	reconcilertestingv2.TestSubscriptionOnK8s(ens.TestEnsemble, sub,
+	CheckSubscriptionOnK8s(g, jsTestEnsemble.Ensemble, sub,
 		testingv2.HaveTypes([]string{testingv2.OrderCreatedEventTypePrefixEmpty}),
 		testingv2.HaveCondition(testingv2.DefaultReadyCondition()),
 		testingv2.HaveSubscriptionReady(),
 	)
 
 	expectedNatsSubscription := []gomegatypes.GomegaMatcher{
-		reconcilertestingv2.BeExistingSubscription(),
-		reconcilertestingv2.BeValidSubscription(),
-		reconcilertestingv2.BeJetStreamSubscriptionWithSubject(testingv2.EventSource,
-			testingv2.OrderCreatedEventTypePrefixEmpty, eventingv1alpha2.TypeMatchingExact, ens.jetStreamBackend.Config),
+		BeExistingSubscription(),
+		BeValidSubscription(),
+		BeJetStreamSubscriptionWithSubject(testingv2.EventSource,
```

Comment:
nit: please move the first argument down one line :)

Reply:
done